src/classes/Passage.ts

```diff
@@ -272,4 +273,32 @@ export default class Passage {
             });
         return magicLinkData;
     }
+
+    /**
+   * Get App Info about an app
+   *
+   * @return {Promise<AppObject>} Passage App object
+   */
+    async getApp(): Promise<AppObject> {
+        if (!this.#apiKey) {
+            throw new Error("A Passage API key is needed to make a getUser request");
+        }
+
+        const appData: AppObject = await axios
+            .get(`https://api.passage.id/v1/apps/${this.appID}`, {
+                headers: {
+                    Authorization: `Bearer ${this.#apiKey}`,
+                },
+            })
```

Comment:
We likely don't need this authorization header since this is an unauthenticated route